src/olmo_core/version.py

```diff
@@ -1,5 +1,5 @@
 _MAJOR = "2"
-_MINOR = "1"
+_MINOR = "2"
```

Comment:
I typically wait until I do an actual release to change these, otherwise it's easy to accidentally increment the version twice.